scummvm_template/index.html.tmpl

```diff
@@ -71,7 +71,7 @@
 </ul>
 	</div> </div> 
 <div><img src="./images/hangmonk.gif" alt="monkey" width="55" height="57" class="monkey float_right" /></div> 
-	<div id="menu_banners"> 
+	<div id="menu_banners" style="font-size: 10px;"> 
 <strong>Last updated:</strong><br>
```

Comment:
This is necessary to keep the small font size of the "Last updated" section in the menu sidebar. Without this, it would be 9pt too which looks not nice IMHO.